modules/test/playwright/tests/frontend-data-set-views-web/visualizationModes.spec.ts

```diff
@@ -435,6 +435,87 @@ test.describe('Visualization Modes in Data Set Manager', () => {
 		});
 	});
 
+	test('Assert that it is possible to change the order of fields in the table using the keyboard', async ({
+		page,
+		visualizationModesPage,
+	}) => {
+		const SAMPLE_SCALAR_FIELD = 'id';
+		const SAMPLE_OBJECT_FIELD = 'fdsViewFDSFieldRelationship';
+		const SAMPLE_OBJECT_CHILD_FIELD = 'id';
+
+		await test.step('Navigate to table visualization mode page', async () => {
+			await visualizationModesPage.goto({
+				dataSetLabel,
+			});
+
+			await visualizationModesPage.selectTab('Table');
+
+			await expect(
+				visualizationModesPage.page.getByPlaceholder('Search')
+			).toBeVisible();
+		});
+
+		await test.step('Add fields', async () => {
+			await visualizationModesPage.openAddFieldsModal();
+
+			await visualizationModesPage.selectField({
+				fieldName: SAMPLE_SCALAR_FIELD,
+			});
+
+			await visualizationModesPage.selectField({
+				dataId: `${SAMPLE_OBJECT_FIELD}.*`,
+				fieldName: SAMPLE_OBJECT_FIELD,
+			});
+
+			await visualizationModesPage.selectField({
+				dataId: `${SAMPLE_OBJECT_FIELD}.${SAMPLE_OBJECT_CHILD_FIELD}`,
+				fieldName: SAMPLE_OBJECT_CHILD_FIELD,
+			});
+
+			await saveFromModal({
+				page,
+			});
+		});
+
```

Comment:
It'd be nice to assert the order of all fields here, and reuse this assertion in the step named "Assert that the field has moved one place up". You can use 2 approaches at least:
 - function taking the array of expected row values and assert compute the locator with 'table.orderable-table > tbody tr:nth-child(i+1)` where `i` tells the position in the array of expected values.
 - similarly but using the table locator as you do [here](https://github.com/liferay-frontend/liferay-portal/pull/4234/commits/83d210bfe6ff282c40aba3cff17fd1d98b1dbd41#diff-4fdd0b230c3151f1fc9543222b2f4adc280e7a7042036293832f82a32d5a0f86R505-R515)

Reply:
Could we simplify even further and merge two tests into one? There is a lot of duplication in the setup, and the `Check if field is added to the last position` assertion is not destructive to UI. After it is successful, it looks like we can keep going with focus and order steps.